clients/go/cmd/zbctl/internal/commands/activateJobs.go

```diff
@@ -62,14 +62,20 @@ var activateJobsCmd = &cobra.Command{
 		}
 
 		jobsCount := len(jobs)
+		var activatedJobs []*pb.ActivatedJob
 		if jobsCount > 0 {
-			if err := printJSON(jobs); err != nil {
-				return err
+			for _, job := range jobs {
+				activatedJobs = append(activatedJobs, job.ActivatedJob)
 			}
 		} else {
-			fmt.Println("[]")
+			activatedJobs = make([]*pb.ActivatedJob, 0)
```

Comment:
I think we don't need to do this since `printJSON` is using the `EmitUnpopulated` option. That will already make the field be marshalled as an empty list

Reply:
Done